### PR TITLE
ModShift cleanup

### PR DESCRIPTION
**Pull request by mustaric**
Fix Modshift Plotting so that it plots two orbits instead of one.
Added Fred calculation. 
Fixed the lightkurve v2 issue, so vetters calls it correctly.

### CODE REVIEW

File: exovetter/vetters.py [hunk 1]
```diff
@@ -84,31 +84,47 @@ def plot(self, tce, lightcurve):
...
     def run(self, tce, lightcurve):
-        """What I want this to look like, but it doesn't work yet
         """
-        time = lightcurve.time
-        flux = lightcurve.flux
-        time_offset_str = lightcurve.time_format
+        run ModShift vetter
+
+        Parameters
+        ---------
```
Comment (pllim) [inline exovetter/vetters.py:96]:
```suggestion
        ----------
```

File: exovetter/vetters.py [hunk 2]
```diff
@@ -84,31 +84,47 @@ def plot(self, tce, lightcurve):
...
+        tce : tce object
+        tce object is a dictionary that contains information about the tce
+        to vet, like period, epoch, duration, depth
```
Comment (pllim) [inline exovetter/vetters.py:99]:
```suggestion
            tce object is a dictionary that contains information about the tce
            to vet, like period, epoch, duration, depth
```

File: exovetter/vetters.py [hunk 3]
```diff
@@ -84,31 +84,47 @@ def plot(self, tce, lightcurve):
...
+
+        lc : lightkurve object
+        lightkurve object with the time and flux to use for vetting.
```
Comment (pllim) [inline exovetter/vetters.py:102]:
```suggestion
            lightkurve object with the time and flux to use for vetting.
```

File: exovetter/vetters.py [hunk 4]
```diff
@@ -84,31 +84,47 @@ def plot(self, tce, lightcurve):
...
+
+        Runs modshift.compute_modeshift_metrics to populate the vetter object.
```
Comment (pllim) [inline exovetter/vetters.py:104]: I can't tell if this is part of `lc` description or what. If this is a general statement, please move it before Parameters.

**Review by pllim — COMMENTED** [review]
Hopefully this suggestion will fix the RTD failure. You can also first build the docs locally to make sure it has no warnings before pushing out the commit: `cd docs; make html`

You can ignore the "dev deps" failure.

**Comment by mustaric** [comment]
I'm abandoning this pull request as it became too painful to try to merge with master.  

Days like these I seriously question why git was implemented the way it was...

**Pull request closed without merging**